invenio_rdm_records/contrib/journal/custom_fields.py

```diff
@@ -12,13 +12,151 @@
 - journal.volume
 """
 
+from abc import ABC
+
 from invenio_i18n import lazy_gettext as _
 from invenio_records_resources.services.custom_fields import BaseCF
-from invenio_records_resources.services.records.facets import CFTermsFacet
-from marshmallow import fields
+from marshmallow import fields, missing
 from marshmallow_utils.fields import SanitizedUnicode
 
 
+class DumpProcessorMixin(ABC):
+    """Mixin class that defines pre_dump and post_dump methods to extend records serialization."""
+
+    def post_dump(self, data, original={}, **kwargs):
```

Comment:
major: do not use mutable objects in func parameters

Reply:
In this case, the original might (or not) be passed by the schema dumper. In marshmallow that is controlled with the `pass_original=True` parameter. That's why I added a default there, to at least make sure it was an object if not passed (and `original.get()` does not fail).

Reply:
```suggestion
    def post_dump(self, data, original=None, **kwargs):
```